src/main/java/com/couchbase/lite/Database.java

```diff
@@ -1280,18 +1280,20 @@ public boolean close() {
         views = null;
 
         // Make all replicators stop and wait:
+        boolean stopping = false;
         synchronized (activeReplicators) {
             for (Replication replicator : activeReplicators) {
                 if (replicator.getStatus() == Replication.ReplicationStatus.REPLICATION_STOPPED)
                     continue;
                 replicator.stop();
+                stopping = true;
             }
 
             // maximum wait time per replicator is 60 sec.
             // total maximum wait time for all replicators is between 60sec and 119 sec.
             long timeout = Replication.DEFAULT_MAX_TIMEOUT_FOR_SHUTDOWN * 1000;
             long startTime = System.currentTimeMillis();
-            while (activeReplicators.size() > 0 &&
+            while (activeReplicators.size() > 0 && stopping &&
```

Comment:
Just a question. After this fix, the recommended MAX timeout is still 60 or can it be reduced to a very small amount?